<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectroscopeTab</class>
 <widget class="QWidget" name="SpectroscopeTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrum analyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCustomPlot" name="plotArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="spectroscopeEnable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Enable spectroscopy</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download from FGPA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset spectrum</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="boxcarTrigger">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setBoxcarTrigger">
         <property name="text">
          <string>Set boxcar trigger</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="useZCDTrigger">
       <property name="text">
        <string>Use ZCD trigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableSpectrumDMA">
       <property name="text">
        <string>Transfer spectra over DMA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plot channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="channelPlotSource">
       <item>
        <property name="text">
         <string>Disabled</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CH1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CH2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CH3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CH4</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Spectrum bin count</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1">
       <item>
        <widget class="QSlider" name="binCount">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>13</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>13</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="binCountLabel">
         <property name="text">
          <string>16384</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="windowDuration">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setWindowDuration">
         <property name="text">
          <string>Set window</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Total count:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pulseRate">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
